src/three/factories/ModelsFactory.js

```diff
@@ -72,25 +72,21 @@ class ModelsFactory {
 
   getTexture$(path) {
     return Observable.create(observer => {
-      new TextureLoader().load(path, value => {
-        observer.next(value);
+      new TextureLoader().load(path, texture => {
+        observer.next(texture);
         observer.complete();
```

Comment:
BTW If we wont to better use TextureLoader and rxjs observable we could do use textureLoader callbacks:
onLoad : Function, onProgress : Function, onError : Function

e.g.:
new TextureLoader().load(path, onLoad => observer.complete(onLoad), onProgress => observer.next(onProgress), onError => observer.error(onError)

Reply:
Please note three.js r84 dropped support for TextureLoader progress events.